plugin-server/src/worker/vm/upgrades/historical-export/export-historical-events.ts

```diff
@@ -47,13 +47,14 @@ export function addHistoricalEventsExportCapability(
         // the historical export to duplicate them
         meta.global.timestampBoundariesForTeam = timestampBoundaries
 
+        console.log('##########', timestampBoundaries)
```

Comment:
keeping these (below too) was deliberate?

Reply:
not at all, thanks for flagging